<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Modify_Pwd</class>
 <widget class="QWidget" name="Admin_Modify_Pwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>259</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Admin_Modify_Pwd
{
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLineEdit" name="lePwd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>85</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>133</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>再次确认密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePwdConfirm">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>确认  </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.check.png</normaloff>:/imgIcon/appbar.check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
	border-radius:5px;
}

QPushButton:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>取消  </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.2.png</normaloff>:/imgIcon/appbar.close.2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>-2</y>
     <width>76</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imgIcon/appbar.cog.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>14</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
	background-color:rgb(50 , 204 , 194);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(50, 184, 179);
}

QPushButton:pressed
{
    background-color:rgb(41, 144, 163);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.png</normaloff>:/imgIcon/appbar.close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>15</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>修改管理员密码</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imageIcon.qrc"/>
 </resources>
 <connections/>
</ui>
